contracts/UniswapV2Factory.sol

```diff
@@ -46,4 +47,15 @@ contract UniswapV2Factory is IUniswapV2Factory {
         require(msg.sender == feeToSetter, 'UniswapV2: FORBIDDEN');
         feeToSetter = _feeToSetter;
     }
+    
+    function setProtocolFee(uint8 _protocolFee) external {
+        require(msg.sender == feeToSetter, 'UniswapV2: FORBIDDEN');
+        require(_protocolFee > 0, 'UniswapV2: FORBIDDEN_FEE');
```

Comment:
Don't we want `_protocolFee >= 0`?

Reply:
Good question, in case you wanna have a 0 protocolFee you can set the feeTo address to 0x. This will skip the fee calculation, which would be the same as having the protocolFee 0 but it wont execute the protocolFee mint each time the liquidity burn/mint. This is why we require the protocolFee to me higher than 0 and we disable/enable it depending on the feeTo receiver. Just as uniswap does.

Reply:
Ahh okay, thanks for the explanation. I'm going to have to improve upon my understanding of the protocol.

Reply:
So why don't we need to specify an upper bound for the protocol fee?

Reply:
Because `feeTo = 0x` is used when you want `protocolFee = 0`
If you remove that require you can still have protocolFee 0, but this is designed in a way that if you want to "disable" the protocolFee you just set the feeTo to 0x

Reply:
> So why don't we need to specify an upper bound for the protocol fee?

yeah I believe this could be an attack vector. For example setting the fee to 100%

Reply:
Apparently Uniswap V.2. Has a limit on the protocol fee they can set.

Reply:
Resolving conversation.